null_safety_examples/cookbook/networking/send_data/pubspec.yaml

```diff
@@ -0,0 +1,13 @@
+name: send_data
+description: Send data
+
+environment:
+  sdk: '>=2.12.0 <3.0.0'
+
+dependencies:
+  flutter:
+    sdk: flutter
+  http: ^0.13.3 
```

Comment:
Done, and a couple of changes were necessary to make the code nicer! I think it would be nice to add it to the rest of extracted examples. What do you think?

cc. @sfshaza2 @RedBrogdon